src/renderer/shared/effector/helpers/series.ts

```diff
@@ -0,0 +1,21 @@
+import { type Effect, type Event, type EventCallable, createEvent, createStore, sample } from 'effector';
+
+import { nonNullable } from '@/shared/lib/utils';
+
+export const series = <T>(target: EventCallable<T> | Effect<T, any>): EventCallable<T[]> => {
```

Comment:
can you add a comment or test for this function so it will be easier to understand how we use it later